mitodl/hugo-course-publisher
# update home page styles

#### Pre-Flight checklist


#### What are the relevant tickets?

part of #314

#### What's this PR do?

this implements the new design for the home template. I didn't see a corresponding mobile design, so I took a stab at a 'column' view to correspond to the 'row' view in the design.

I also removed the React component we were using for the home search page - turns out it's possible to implement the same behavior with just a `<form>` and a pair of `<input>` tags! nice.

#### How should this be manually tested?

check out the mockups and the design, and make sure it looks legit. try out the home page search box too, and make sure that works as it should.


#### Screenshots (if appropriate)
![Screenshot from 2020-11-13 12-05-35](https://user-images.githubusercontent.com/6207644/99099617-b49f3c00-25a8-11eb-98b7-67b803158e89.png)
![Screenshot from 2020-11-13 12-05-27](https://user-images.githubusercontent.com/6207644/99099619-b537d280-25a8-11eb-98bc-62d6dbcadc62.png)
![Screenshot from 2020-11-13 12-05-17](https://user-images.githubusercontent.com/6207644/99099621-b537d280-25a8-11eb-8e1c-727d727cf415.png)
![Screenshot from 2020-11-13 12-05-07](https://user-images.githubusercontent.com/6207644/99099622-b5d06900-25a8-11eb-8dc1-71805e89e7f1.png)
